test/e2e/batcher/test_batcher_custom_port.py

```diff
@@ -0,0 +1,79 @@
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+from kubernetes import client
+
+from kserve import KServeClient
+from kserve import constants
+from kserve import V1beta1PredictorSpec
+from kserve import V1beta1Batcher
+from kserve import V1beta1TorchServeSpec
+from kserve import V1beta1InferenceServiceSpec
+from kserve import V1beta1InferenceService
+
+from kubernetes.client import V1ResourceRequirements, V1ContainerPort
+from ..common.utils import predict
+from ..common.utils import KSERVE_TEST_NAMESPACE
+from concurrent import futures
+
+kserve_client = KServeClient(config_file=os.environ.get("KUBECONFIG", "~/.kube/config"))
+
+
+def test_batcher_custom_port():
+    service_name = 'isvc-pytorch-batcher-custom'
+    predictor = V1beta1PredictorSpec(
+        batcher=V1beta1Batcher(
+            max_batch_size=32,
+            max_latency=5000,
+        ),
+        min_replicas=1,
+        pytorch=V1beta1TorchServeSpec(
+            args=["--http_port=5000"],
```

Comment:
I think the torchserve actually does not support this argument, we can change to use sklearnserver instead.

Reply:
Yes, made changes to use `env=[V1EnvVar(name='TS_INFERENCE_ADDRESS', value='"http://127.0.0.1:5000"')]`
but it only works if config.properties entry is added to use environment variables... Let me see if there is another way, if not use sklearn...